packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -28,6 +30,10 @@ class OpPerfTelemetry {
     private opSendTimeForLatencyStatistics: number | undefined;
     private clientSequenceNumberForLatencyStatistics: number | undefined;
 
+    private opTimeSittingInInboundQueue: number | undefined;
+    private durationRoundtripTime: number | undefined;
+    private durationSittingInInboundQueue: number | undefined;
```

Comment:
All of these values need to be reset to undefined on disconnect. This ensures we could do more strict asserts (as I suggested below) and thus better control state transitions.